lib/segment/src/vector_storage/quantized/quantized_scorer_builder.rs

```diff
@@ -172,4 +190,68 @@ impl<'a> QuantizedScorerBuilder<'a> {
             }
         }
     }
+
+    fn new_multi_quantized_scorer<TElement, TMetric, TEncodedQuery>(
```

Comment:
the same as `new_quantized_scorer` but for multivectors. It uses `QuantizedScorer::new_multi` inside